universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRendererFactory.java

```diff
@@ -1,9 +1,22 @@
 package views.questiontypes;
 
+import services.applicant.ApplicantData;
 import services.applicant.ApplicantQuestion;
+import services.question.QuestionDefinition;
+import services.question.QuestionDefinitionBuilder;
+import services.question.QuestionType;
+import services.question.UnsupportedQuestionTypeException;
 
 public class ApplicantQuestionRendererFactory {
 
+  public ApplicantQuestionRenderer getSampleRenderer(QuestionType questionType)
+      throws UnsupportedQuestionTypeException, UnsupportedOperationException {
```

Comment:
Question (non-blocking): Is `UnsupportedOperationException` necessary?

Reply:
Done.